roles/wildfly_utils/tasks/jboss_cli.yml

```diff
@@ -9,6 +9,10 @@
     fail_msg: "Missing required parameters to execute JBoss CLI."
     quiet: true
 
+- name: "Determine CLI port (if port offset is used: {{ wildfly_port_range_offset }})"
+  ansible.builtin.set_fact:
```

Comment:
Do not set a fact overriding a user value; set a new var, or just evaluate directly on line 20 below `port: "{{ (jboss_cli_controller_port | int) + (wildfly_port_range_offset | int) }}` (no default since jboss_cli_controller_port is always defined)

Reply:
Right, I intented to change that indeed, but in the battle to "make it work", it got lost. Nice catch, thanks!